tasks/copy_files_to_archive/test/unit_tests/test_copy_files_to_archive.py

```diff
@@ -6,336 +6,268 @@
 import os
 import unittest
 import uuid
+from random import randint
+from unittest import mock
```

Comment:
Possibly create a card for later work to possibly use moto (https://github.com/spulec/moto) for mocking AWS functionality. This is what Rizbi was using for some of his unit tests. Thoughts?

Reply:
Eh. I can see the appeal, but I don't know if it's worth redoing existing tests. We'd need to use both anyway, since some tests check call_count.